### PR TITLE
Plone 5.1.x Support (NG)

### PR DESCRIPTION
**Pull request by jone**
Replaces / closes https://github.com/4teamwork/ftw.testbrowser/pull/47

Introduce support for Plone 5.1.
- requires https://github.com/4teamwork/ftw.testing/pull/32 for integration testing with the traversal driver
- extends the traversal driver to support plone.protect's auto CSRF protection
- various fixes for page objects because of changes in Plone
- see separate commits for details

### CODE REVIEW

File: sources.cfg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    mr.developer
+
+auto-checkout =
+# https://github.com/4teamwork/ftw.testing/pull/32/files
+    ftw.testing
+
+
+[branches]
+# https://github.com/4teamwork/ftw.testing/pull/32/files
+ftw.testing = jone-integration-testing
```
Comment (jone) [inline sources.cfg]: 🚧 https://github.com/4teamwork/ftw.testing/pull/32

File: sources.cfg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[buildout]
+extends =
+    http://kgs.4teamwork.ch/sources.cfg
+
+extensions +=
+    mr.developer
+
+auto-checkout =
+# https://github.com/4teamwork/ftw.testing/pull/32/files
+    ftw.testing
```
Comment (jone) [inline sources.cfg]: 🚧 https://github.com/4teamwork/ftw.testing/pull/32

File: development.cfg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    test-plone-4.3.x.cfg
+    test-plone-5.1.x.cfg
```
Comment (lukasgraf) [inline development.cfg]: I would also update the Trove classifiers in `setup.py` to declare Plone 5 compatibility

Comment (jone) [reply development.cfg]: Ah, yes, sure 👍

**Comment by jone** [comment]
Thanks for the reviews!

**Pull request merged**